Realme/RE513CL1/res/xml/power_profile.xml

```diff
@@ -1,36 +1,12 @@
 <?xml version="1.0" encoding="utf-8"?>
 <device name="Android">
-    <item name="ambient.on">0.1</item>
-    <item name="screen.on">0.1</item>
-    <item name="screen.full">0.1</item>
-    <item name="bluetooth.active">0.1</item>
-    <item name="bluetooth.on">0.1</item>
-    <item name="wifi.on">0.1</item>
-    <item name="wifi.active">0.1</item>
-    <item name="wifi.scan">0.1</item>
-    <item name="audio">0.1</item>
-    <item name="video">0.1</item>
-    <item name="camera.flashlight">0.1</item>
-    <item name="camera.avg">0.1</item>
-    <item name="gps.on">0.1</item>
-    <item name="radio.active">0.1</item>
-    <item name="radio.scanning">0.1</item>
-    <array name="radio.on">
-        <value>0.2</value>
-        <value>0.1</value>
-    </array>
+    <item name="battery.capacity">3000</item>
```

Comment:
https://www.gsmarena.com/realme_8_5g-10846.php says this is wrong

Reply:
```
:/ # cat /sys/class/power_supply/battery/charge_full
4000
:/ #
```
it's actually 4000 mAh and not 5000 or 3000 😐